disassemblers/ofrak_ghidra/ofrak_ghidra/ghidra_model.py

```diff
@@ -82,6 +87,38 @@ class OfrakGhidraMixin:
     `OfrakGhidraScript` member of the class.
     """
 
+    async def fixup_address(self, va: int, resource: Resource) -> int:
```

Comment:
I think calling this on every ComplexBlock and BasicBlock will have a very heavy performance penalty. See my other comment on not using the `va_addend`, we should only need to do this fixup up in the `CodeRegionUnpacker`, nowhere else.

Reply:
I agree it's bad for performance. That's why we fix OFRAK's CodeRegion's in the CodeRegionUnpacker or earlier